apps/frontend/src/api/API.ts

```diff
@@ -33,15 +33,12 @@ export default class API {
       Accept: 'application/json; version=1.0.1',
     }
   }
-  async getRequestFullUrl(
-    url: string,
-    params: Record<string, string | number | boolean>,
-    suppressError: boolean,
-  ) {
+
+  async getRequestFullUrl(url: string, params: Record<string, unknown>, suppressError: boolean) {
     return axios
       .get(url, {
-        data: {},
-        params: params,
+        params,
+        paramsSerializer: (params) => qs.stringify(params, { arrayFormat: 'repeat' }),
```

Comment:
@YusraSeyam this is where the error is coming from. From the build logs:

```
src/api/API.ts:41:39 - error TS2304: Cannot find name 'qs'.

41         paramsSerializer: (params) => qs.stringify(params, { arrayFormat: 'repeat' }),
                                         ~~
```

Where did you come up with this line of code? If it's from an example somewhere, I'd guess you need an extra import or something similar.